install/installer/pkg/components/ws-manager/configmap.go

```diff
@@ -88,7 +88,7 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 
 	var schedulerName string
 	gitpodHostURL := "https://" + ctx.Config.Domain
-	workspaceClusterHost := fmt.Sprintf("ws.%s", ctx.Config.Domain)
+	workspaceClusterHost := fmt.Sprintf("ws%s.%s", installationShortNameSuffix, ctx.Config.Domain)
```

Comment:
@sagor999 I think with this change, we can remove `ucfg.Workspace.WorkspaceClusterHost`, `ucfg.Workspace.WorkspaceURLTemplate`, `ucfg.Workspace.WorkspaceURLTemplate`, `ucfg.Workspace.HostURL`